src/indexer/utils/substrate/index.ts

```diff
@@ -116,13 +118,13 @@ export async function saveDeposit(
   if (transfer) {
     const dataTransferToUpdate = {
       depositNonce: Number(depositNonce),
-      sender,
       amount: decodedAmount,
       resourceID: resourceId,
       fromDomainId: originDomainId.toString(),
       toDomainId: destinationDomainId,
       timestamp: timestamp,
       destination: `0x${depositData.substring(2).slice(128, depositData.length - 1)}`,
+      sender: transfer.accountId!,
```

Comment:
I think this could be null | undefined if the executionTransfer is inserted first, there will be no accountId

Reply:
you were rigth about this. I have made de correction